test/test.js

```diff
@@ -85,4 +87,37 @@ describe('IDBExportImport', function() {
       assert.ifError(e);
     });
   });
+  it('Should import and export the database with composite keys and empty keys', function(done) {
```

Comment:
I don't understand what "composite keys" means here.

Does it refer to multiple stores? (colors, shapes, color_shape, empty)

Or does it refer to the data in the stores? Like `color_shape: '[shape+color]',`

Reply:
Improved the test description